src/components/Home/styles/style.scss

```diff
@@ -0,0 +1,77 @@
+.container {
+    //home styling
```

Comment:
we don't need these comments always name class like that

Reply:
done